code/mob/living/carbon/human.dm

```diff
@@ -1430,7 +1430,11 @@
 				src.say_language = original_language
 				return
 
-	message = process_accents(src,message)
+	if (src.wear_mask && istype(src.wear_mask,/obj/item/clothing/mask/mime/cursed))
+		whisper(message, forced=TRUE)
```

Comment:
this looks like it makes accents not apply?

Reply:
accents are processed in whisper too. whisper is basically say 2......

Reply:
oh shit i elsed it heh...

Reply:
whisper code:
![Code_94YJ3cUBVH](https://user-images.githubusercontent.com/12144569/109736511-ea285f00-7b70-11eb-81ef-ce35ac324fcb.png)

Reply:
i'm basically redirecting say to whisper and cutting it.